test/cookbooks/test/recipes/default.rb

```diff
@@ -17,6 +17,10 @@
 # limitations under the License.
 #
 
+if %w(debian ubuntu).include? node['platform']
```

Comment:
if platform_family?('debian') would do the same thing here

Reply:
yea i actually just updated to that, but turns out this didnt help the travis build anyway so i might remove it. 

not sure why i keep getting this on `default-debian-8`

```
Recipe: java::openjdk
  * apt_package[openjdk-8-jdk] action install
    * No candidate version available for openjdk-8-jdk
    ================================================================================
    Error executing action `install` on resource 'apt_package[openjdk-8-jdk]'
    ================================================================================
    
    Chef::Exceptions::Package
    -------------------------
    No candidate version available for openjdk-8-jdk
```

Reply:
it ust just not be available on debian 8 yet, its not tested there in the java cookbook https://github.com/agileorbit-cookbooks/java/blob/master/.kitchen.yml#L48